mlir/include/mlir/Dialect/MIOpen/Tuning/GridwiseGemmParams.h

```diff
@@ -398,8 +398,8 @@ class PopulateParamsBase {
 };
 
 struct InitParamsNonXDL : InitParams, Serializable<InitParamsNonXDL> {
-  InitParamsNonXDL(int64_t mPerBlock, int64_t nPerBlock, int64_t kPerBlock,
-                   int64_t mPerThread, int64_t nPerThread, int64_t bSize)
+  InitParamsNonXDL(int64_t bSize, int64_t mPerBlock, int64_t nPerBlock,
```

Comment:
@jerryyin what's the reason to move `bSize` to the front? sync with logic in MIOpen? what about XDLOPS route?

Reply:
The ordering doesn't matter. I change it such that it is easier to compare the list of init parameters to the ones in MIOpen.

Reply:
ok.